src/js/events/CosmosPackagesActions.js

```diff
@@ -149,14 +187,65 @@ const CosmosPackagesActions = {
     });
   },
 
+  fetchServiceDescription(serviceId) {
+    RequestUtil.json({
+      contentType: getContentType({
+        action: "describe",
+        actionType: "request",
+        entity: "service",
+        version: "v1"
+      }),
+      headers: {
+        Accept: getContentType({
+          action: "describe",
+          actionType: "response",
+          entity: "service",
+          version: "v1"
+        })
+      },
+      method: "POST",
+      url: `${Config.rootUrl}/cosmos/service/describe`,
+      data: JSON.stringify({ appId: serviceId }),
+      success(response) {
+        const cosmosPackage = response.package;
+
+        AppDispatcher.handleServerAction({
+          type: REQUEST_COSMOS_SERVICE_DESCRIBE_SUCCESS,
+          data: cosmosPackage,
+          serviceId
```

Comment:
Let's make sure we test `serviceId` is emitted on success and error.

Reply:
Thanks, added this 👍